examples/nuxt-3/pages/collections/[handle].vue

```diff
@@ -42,17 +42,17 @@ const { data: collectionPageData, error: pageFetchError } = await useAsyncData(
       const {
         productConnection: {
           pageInfo: productPaginationInfo,
-          totalProductCount,
+          productCount,
           edges: productEdges
         },
-        ...collection
-      } = allProductCollections.edges[0]?.node;
-      const products = productEdges.map((edge) => edge.node);
+        ...productCollection
+      } = allProductCollections.edges[0].node;
+      const collectionProducts = productEdges.map((edge) => edge.node);
```

Comment:
It's worth calling out that this still errors  if allProductCollections.edges[0] is undefined. It just doesn't have any eslint hints since it's untyped (which might be okay).

Reply:
Come to think of it, since we're in a:
```js
if (allProductCollections.edges[0]) {
```
, we should be safe.

Reply:
Good catch!